src/app.py

```diff
@@ -1,4 +1,21 @@
 import pygame
 
 def main():
-    return
+     
+    pygame.init()
+    
+    pygame.display.set_logo()
+    pygame.display.set_caption("Kopi-Cat Uno")
+
+    screen = pygame.display.set_mode((720,480))
```

Comment:
I think after our game is nearly finished, we should have our game auto full screen. For now, we can create a commented line of code that does that so that we can continue to test our code with the small window.

Reply:
not everyone prefers to play full-screen so we can make it such that they can set it full screen manually. Except, I'm not sure how to do that yet. Unless, it's already like a default thing

Reply:
@icesotope41, we can reference from here https://stackoverflow.com/questions/31538506/how-do-i-maximize-the-display-screen-in-pygame

Reply:
Hmm...maybe at we could center the window?

Reply:
But I feel its more natural to have the game auto full screen